CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 This document outlines major changes between releases.
 
+## [Unreleased]
+
+### Added
+- Exposure of pool metrics (#615)
+### Changed
+- Improved wallet configuration via `.yaml` config and environment variables (#607)
+### Removed
+### Fixed
```

Comment:
I think we can add new section when we need it. Also we should add `Updating from v0.23.0` section, see https://github.com/nspcc-dev/neofs-s3-gw/blob/master/CHANGELOG.md#updating-from-v0220

Reply:
I suppose adding of the most used sections in advance will make further appending of new changes a little more friendly. We can remove unused sections when we release.

Reply:
Also core dev team fills changelog in the same way: https://github.com/nspcc-dev/neofs-node/blob/master/CHANGELOG.md